README.md

```diff
@@ -9,11 +9,19 @@ This information will come along with the template, so __you may just want to ge
 
 # To get started immediately: #
 
+If you're using `sbt 1.2.X`, execute the following when you want to start a new scala-sbt project:
+
 ```bash
 sbt new sbchapin/quickstart-scala-sbt-native-image.g8
 ```
 
-Execute the preceding `sbt` command wherever you want to start a new scala-sbt project.  **It will create a new folder in your active directory.**
+As a result of [this issue](https://github.com/sbt/sbt/issues/5063), if you're using sbt `1.3.0` use this command instead:
+
+```bash
+sbt --supershell=false new sbchapin/quickstart-scala-sbt-native-image.g8
+```
+
+Running either one of these commands **will create a new folder in your active directory.** 
```

Comment:
I hate how this convolutes the start of your README, which used to be so clean and nice. The output from:
```bash
sbt --supershell=false new sbchapin/quickstart-scala-sbt-native-image.g8
```
looks really good still, so another option might be to just remove all this bullshit I added and cherry pick just the new command.

Reply:
I used to think more documentation, more better.  Now I think documentation density. Tomorrow I'll think I should have thought less.  

Today, my suggestion is to replace the english with comments:
```bash
# sbt < 1.3.0
sbt new sbchapin/quickstart-scala-sbt-native-image.g8
# sbt >= 1.3.0 (https://github.com/sbt/sbt/issues/5063)
sbt --supershell=false new sbchapin/quickstart-scala-sbt-native-image.g8
```